VimeoNetworking/VimeoNetworking/AuthenticationController.swift

```diff
@@ -227,6 +227,20 @@ final public class AuthenticationController
         self.authenticate(request: request, completion: completion)
     }
     
+    /**
+     Execute a constant token grant request. This type of authentication allows access to public and personnal content on Vimeo. Constant token are usually generated for API apps see https://developer.vimeo.com/apps
+     
+     - parameter completion: handles authentication success or failure
+     */
+    public func accessToken(token: String, completion: AuthenticationCompletion)
+    {
+        let customSessionManager =  VimeoSessionManager.defaultSessionManager(accessTokenProvider: {() -> String? in return "\(token)"})
```

Comment:
Small thing, we can actually simplify

`{() -> String? in return "\(token)"}`

to just

`{ token }`